graylog2-web-interface/src/components/datanode/migrations/InPlaceMigration.tsx

```diff
@@ -67,12 +62,10 @@ const StyledPanelGroup = styled(PanelGroup)`
   }
 `;
 
-const InPlaceMigration = ({ currentStep, onTriggerNextStep }: Props) => {
+const InPlaceMigration = ({ currentStep, onTriggerStep }: MigrationStepComponentProps) => {
```

Comment:
there is an issue When I go to In-Place migration
```suggestion
Uncaught (in promise) TypeError: onTriggerStep is not a function
    at onStepComplete (InPlaceMigration.tsx:68:1)
```